modules/document_repository/README.md

```diff
@@ -0,0 +1,30 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a useful tool that provides studies with a centralized location for important documents. Files may be uploaded and organized under any user-defined category. Subcategories can also be defined by the user to be nested under parent categories.
+
+## Intended Users
+ 
+ All Loirs user could view, update, edit and delete the upload files with right permissions.
+
+## Scope
+
+ For all of the Loris users.
+
+## Permissions
+
+ A user has "document_repository_view" permission could view and upload files in Document Repository.
```

Comment:
missing the word "that" before "has" and "the" after ("A user that has the [..]") and "can view", not "could view" (same comments apply to the delete permission below)